masonry/src/contexts.rs

```diff
@@ -403,6 +415,34 @@ impl<'a> MutateCtx<'a> {
     }
 }
 
+// --- MARK: WIDGET_REF ---
+// Methods to get a child WidgetRef from a parent.
+impl<'w> QueryCtx<'w> {
+    /// Return a [`WidgetRef`] to a child widget.
+    pub fn get(&self, child: WidgetId) -> WidgetRef<'_, dyn Widget> {
+        let child_state = self
+            .widget_state_children
+            .get_child(child.to_raw())
+            .expect("get: child not found");
+        let child = self
+            .widget_children
+            .get_child(child.to_raw())
+            .expect("get: child not found");
```

Comment:
```suggestion
    pub fn get(self, child: WidgetId) -> WidgetRef<'w, dyn Widget> {
        let child_state = self
            .widget_state_children
            .into_child(child.to_raw())
            .expect("get: child not found");
        let child = self
            .widget_children
            .into_child(child.to_raw())
            .expect("get: child not found");
```

Reply:
Thanks, `ArenaRefChildren::into_child` was the piece I missed when first playing around with these lifetime issues.